CHANGELOG.md

```diff
@@ -0,0 +1,65 @@
+# Changelog
+
+Darts is still in an early development phase and we cannot always guarantee backwards compatibility. Changes suceptible to **break existing code** working on a previous release are marked with a "&#x1F534;". Sorry for the inconvenience!
+
+## [Unreleased](https://github.com/unit8co/darts/tree/develop)
+
+[Full Changelog](https://github.com/unit8co/darts/compare/0.3.0...develop)
+
+## [0.3.0](https://github.com/unit8co/darts/tree/0.3.0) (2020-10-01)
+
+[Full Changelog](https://github.com/unit8co/darts/compare/0.2.3...0.3.0)
+
+### For users of the library:
+**Added:**
+
+- Indexing on TimeSeries [\#150](https://github.com/unit8co/darts/pull/150)
+- New `FourTheta` forecasting model [\#123](https://github.com/unit8co/darts/pull/123), [\#156](https://github.com/unit8co/darts/pull/156)
+- `map()` method for TimeSeries [\#121](https://github.com/unit8co/darts/issues/121), [\#166](https://github.com/unit8co/darts/pull/166)
+- Further improved Multivariate TimeSeries capabilities:
+  - Added multivariate support to the Neural Network based forecasting models, namely `TorchForecastingModel` and its subclasses (`TCNModel` and `RNNModel`)
+  - Added multivariate support to backtesting functions [\#111](https://github.com/unit8co/darts/pull/111)
+- Custom style for matplotlib plots [\#191](https://github.com/unit8co/darts/pull/191)
+- Option to pass a single string as value column name to the `TimeSeries.from_dataframe()` function. [\#114](https://github.com/unit8co/darts/pull/114)
+- sMAPE metric [\#129](https://github.com/unit8co/darts/pull/129)
+- `freq` parameter for TimeSeries generating functions [\#157](https://github.com/unit8co/darts/pull/157)
+- Option to specify a `random_state` at model creation using the `@random_method` decorator on models using neural networks to allow reproducibility of results [\#118](https://github.com/unit8co/darts/pull/118)
+
+**Changed:**
+
+- &#x1F534; **Refactored backtesting** [\#184](https://github.com/unit8co/darts/pull/184)
+  - Moved backtesting functionalities inside `ForecastingModel` and `RegressionModel`
+    - Old syntax: `backtest_forecasting()` and `backtest_regression()`
```

Comment:
What do you think about putting the `model` into the arguments of the old syntax? Actually in that case we might have to use 2 variables: `forecasting_model` and `regression_model`. Also, we could add `*args` and `**kwargs` into the brackets, just to show that everything else stays the same.

Reply:
Great suggestion, I'm changing it